docs/LanguageGuide.md

```diff
@@ -69,6 +69,36 @@ foo ; symbol
 (defmodule <name> <definition1> <definition2> ...) ;; The main way to organize your program into smaller parts
 ```
 
+### Conditional statements with 'cond'
+The cond statement executes a block of code if a specified condition is true. If the condition is false, another block of code can be executed. Examples are given below:
+
+```clojure
+(defndynamic cond-internal [xs]
+  (if (= (length xs) 0)
+    (list)
+    (if (= (length xs) 2)
+      (macro-error "cond has even number of branches; add an else branch")
+      (if (= (length xs) 1)
+        (car xs)
+        (list
+         'if
+         (car xs)
+         (cadr xs)
+         (cond-internal (cddr xs)))))))
+
+(defmacro cond [:rest xs]
+  (cond-internal xs))
+
+```
+Here's an example to print a statement depending on whether it is < or > 10
```

Comment:
Some sort of punctuation at the end of the sentence might be good!

Reply:
Done! :)